rpcs3/util/media_utils.h

```diff
@@ -77,8 +77,8 @@ namespace utils
 		std::vector<u8> data;
 		atomic_t<u64> m_size = 0;
 		atomic_t<u64> duration_ms = 0;
-		atomic_t<bool> track_fully_decoded{false};
-		atomic_t<bool> track_fully_consumed{false};
```

Comment:
I don't like this. Can't you make it opaque to the user with some template magic?

Reply:
I don't think it's possible to change atomic bool to use 32-bit storage without breaking some PS3 struct (possibly in future as well)